desc/plotting.py

```diff
@@ -1593,16 +1593,15 @@ def plot_boundary(eq, phi=None, plot_axis=True, ax=None, return_data=False, **kw
     phi = parse_argname_change(phi, kwargs, "zeta", "phi")
 
     figsize = kwargs.pop("figsize", None)
-    cmap = kwargs.pop("cmap", "rainbow")
+    cmap = kwargs.pop("cmap", "hsv")
```

Comment:
This is a cyclic colormap, which I prefer because it makes it more clear which cross-sections correspond to which toroidal angles.

Reply:
I think twilight or twilight_shifted would be preferred over hsv, it looks better in grayscale and for colorblindness 
https://matplotlib.org/stable/tutorials/colors/colormaps.html

Reply:
I take that back, `twilight` looks pretty ugly. But I'm also really not a fan of `hsv`. Might be a good chance to try some of the cool ones in https://github.com/holoviz/colorcet

Reply:
or if we're already going to start using plotly for 3d stuff, https://plotly.com/python/builtin-colorscales/#builtin-cyclical-color-scales

Reply:
@f0uriest You can update the default colormap if you want to, otherwise this PR is done. 

Honestly I don't think `hsv` looks that different from `rainbow` that we had before. Also it's just a default; you can always supply your own colormap.